dnf/rpm/miscutils.py

```diff
@@ -65,16 +65,24 @@ def getSigInfo(hdr):
     """checks signature from an hdr hand back signature information and/or
        an error code"""
 
-    string = '%|DSAHEADER?{%{DSAHEADER:pgpsig}}:{%|RSAHEADER?{%{RSAHEADER:pgpsig}}:{%|SIGGPG?{%{SIGGPG:pgpsig}}:{%|SIGPGP?{%{SIGPGP:pgpsig}}:{(none)}|}|}|}|'
+    error = 101
+    sigtype = 'MD5'
+    sigdate = 'None'
+    sigid = 'None'
+
+    string = '%|DSAHEADER?{%{DSAHEADER:pgpsig}}:{%|RSAHEADER?{%{RSAHEADER:pgpsig}}:' \
+             '{%|SIGGPG?{%{SIGGPG:pgpsig}}:{%|SIGPGP?{%{SIGPGP:pgpsig}}:{(none)}|}|}|}|'
     siginfo = hdr.sprintf(string)
+
     if siginfo != '(none)':
-        error = 0
-        sigtype, sigdate, sigid = siginfo.split(',')
-    else:
-        error = 101
-        sigtype = 'MD5'
-        sigdate = 'None'
-        sigid = 'None'
+        pos = siginfo.find(',')
```

Comment:
Miscutils.py had to be changed.
Because datetime in some locales contains comma (eg. Czech) or more commas (eg. Polish).
DNF fails for these locales without this commit. 
Examples:
English (no comma): Tue 16 May 2017 02:09:54 PM CEST
Czech (one comma): Út 16. květen 2017, 14:09:54 CEST
Polish (two commans): wto, 16 maj 2017, 14:09:54